### PR TITLE
Adding c-suite blog post

### PR DESCRIPTION
**Pull request by psschwei**
## Proposed Changes 

Adds the "Highlighting the Value of Knative for the C-Suite" blog post, as requested by @omerbensaadon 

Note: I had to fiddle with the formatting for the quotes on the left side, but it is very possible that I did not do so in the optimal way. This is a roundabout way of me saying that someone should definitely take a look at the HTML because there's a very real chance I did something ~stupid~ non-optimal when rendering it.

### CODE REVIEW

**Comment by csantanapr** [comment]
what do you think about adding a small avatar picture next to the name in the quote?
I guess we can get the picture from Twitter avatar
image
cc @omerbensaadon

**Comment by psschwei** [comment]
> what do you think about adding a small avatar picture next to the name in the quote? 

Do we need to get each person's consent to use their likeness? 

If we do decide to add avatars, implementing it in a aesthetically-pleasing way is probably a bit beyond my capabilities...

**Comment by csantanapr** [comment]
ok then, let's ship this and see we can add the avatars later

**Comment by csantanapr** [comment]
@psschwei Can we add this post as a feature post?
image

**Comment by psschwei** [comment]
@csantanapr added

File: blog/docs/articles/highlighting-value-knative-c-suite.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+---
+
+# Highlighting the Value of Knative for the C-Suite
+
+<table style="border: 0;">
+<tr style="background-color: var(--md-default-bg-color);">
+<td style="border: 0;" width="33%">
+
+<div style="padding:50px;">
+<em style="color:#0071c7;font-size: 1em;">"What we are doing with Knative is to provide a developer experience that makes it easier to focus on code. Cloud,developers focus on the business problems they are solving without having to coordinate or wait for approvals from platform teams to scale their apps.. Knative is a framework that helps automate platform capabilities so your apps can scale as if they were running on Serverless compute."</em>
```
Comment (csantanapr) [inline blog/docs/articles/highlighting-value-knative-c-suite.md:16]:
```suggestion
<em style="color:#0071c7;font-size: 1em;">"What we are doing with Knative is to provide a developer experience that makes it easier to focus on code. Cloud developers focus on the business problems they are solving without having to coordinate or wait for approvals from platform teams to scale their apps.. Knative is a framework that helps automate platform capabilities so your apps can scale as if they were running on Serverless compute."</em>
```

File: blog/docs/articles/highlighting-value-knative-c-suite.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+<h3 style="font-weight: bold;">Realize the promise of DevOps.</h3>
+
+<p>For more than ten years, DevOps has been a promise of an effective application development process that deploys faster with less bugs. Offering immense capability but introducing significant complexity, Kubernetes is a key link in realizing value from DevOps. To achieve value at scale, cluster administrators need specialized knowledge, and in the past developers did as well. Knative provides a serverless experience that greatly reduces the need for those who develop and deploy code to have Kubernetes expertise. Operators get consistent, manageable containers when possible and the flexibility to deploy any image when necessary.
+
+<p><i>"What we are doing with Knative is to provide a developer experience that makes it easier to focus on code. Cloud,developers focus on the  business problems they are solving without having to coordinate or wait for approvals from platform teams to scale their apps.. Knative is a framework  that helps automate platform capabilities so your apps can scale as if they were running on Serverless compute."</i>
```
Comment (csantanapr) [inline blog/docs/articles/highlighting-value-knative-c-suite.md:76]:
```suggestion
<p><i>"What we are doing with Knative is to provide a developer experience that makes it easier to focus on code. Cloud, developers focus on the business problems they are solving without having to coordinate or wait for approvals from platform teams to scale their apps.. Knative is a framework  that helps automate platform capabilities so your apps can scale as if they were running on Serverless compute."</i>
```

File: blog/docs/articles/highlighting-value-knative-c-suite.md (hunk 1 above)
Comment (joelbyronbarker) [reply blog/docs/articles/highlighting-value-knative-c-suite.md]: I suggest making the quotes more visually separate from the other content, such as how the quotes like _"Knative gives us a foundation of consistency," said Andrew Webber, Senior Software Engineer._ appears in https://knative.dev/docs/about/case-studies/deepc/

**Comment by csantanapr** [comment]
Thank you @psschwei 🙏 
/approve
/lgtm

**Pull request merged**